main/test/src/mill/main/MainModuleTests.scala

```diff
@@ -161,7 +161,7 @@ object MainModuleTests extends TestSuite {
       test("command") {
         val Left(Result.Failure(failureMsg, _)) = evaluator.evalTokens("show", "helloCommand")
         assert(
-          failureMsg.contains("Expected Signature: helloCommand"),
+          failureMsg.contains("Expected Signature: hello-command"),
```

Comment:
Note that this change to the error message does not mean the original `helloCommand` syntax stops working; the test (below) continues to use `helloCommand`. 

With mainargs 0.6.0, both `helloCommand` and `hello-command` are accepted, but we put the dashed version `hello-command` in the help message by default (unless a different nameMapper is given)